package.json

```diff
@@ -32,12 +34,13 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.19.2",
-    "form-data": "^3.0.0",
-    "fs": "0.0.1-security"
+    "form-data": "^3.0.0"
```

Comment:
We should pin the dependencies. This is a npm package so when people install it, it's better that we know exactly what the dependencies versions are. ("to pin" means to install them without the `^` range marker - you can either remove it manually and run `npm install` to update `package-lock.json` or call `npm i axios@0.19.2 form-data@3.0.0` to install specific versions of libraries).